core/src/test/java/org/apache/calcite/test/SqlToRelConverterTest.java

```diff
@@ -4050,6 +4050,15 @@ private Tester getExtendedTester() {
     sql(sql).ok();
   }
 
+  @Test void testCoalesceOnLieralsJoinNull() {
```

Comment:
correct the test case name to something like "testCoalesceOnUnionOfLiteralsAndNull"